StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1015ClosingGenericBracketsMustBeSpacedCorrectly.cs

```diff
@@ -86,27 +87,13 @@ private void HandleGreaterThanToken(SyntaxTreeAnalysisContext context, SyntaxTok
                 return;
             }
 
-            bool precededBySpace;
-            bool firstInLine;
-
-            bool followedBySpace;
-            bool lastInLine;
+            bool firstInLine = token.IsFirstInLine();
+            bool lastInLine = token.IsLastInLine();
+            bool precededBySpace = firstInLine ? true : token.IsPrecededBySpace();
```

Comment:
You could also write this like `precededBySpace = firstInLine || token.IsPrecededBySpace();`